src/memory.jl

```diff
@@ -147,9 +154,10 @@ const HOSTREGISTER_IOMEMORY = CUDAdrv.CU_MEMHOSTREGISTER_IOMEMORY
     Mem.register(HostBuffer, ptr::Ptr, bytesize::Integer, [flags])
 
 Page-lock the host memory pointed to by `ptr`. Subsequent transfers to and from devices will
-be faster, and can be executed asynchronously. If the `MEMHOSTREGISTER_DEVICEMAP` flag is
-specified, the buffer will also be accessible directly from the GPU. These accesses are
-direct, and go through the PCI bus.
+be faster, and can be executed asynchronously. If the `HOSTREGISTER_DEVICEMAP` flag is
+specified, the buffer will also be accessible directly from the GPU. If the
+`HOSTREGISTER_PORTABLE` flag is specified, any CUDA context can access the memory.
+These accesses are direct, and go through the PCI bus.
```

Comment:
HOSTREGISTER_DEVICEMAP accesses go over the PCI bus, not HOSTREGISTER_PORTABLE ones, so I'd move that sentence.